kube-flagd-proxy/cmd/start.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/open-feature/flagd/core/pkg/logger"
+	"github.com/open-feature/flagd/core/pkg/service"
+	syncServer "github.com/open-feature/flagd/core/pkg/service/sync"
+	sync_store "github.com/open-feature/flagd/core/pkg/sync-store"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap/zapcore"
+)
+
+// start
+
+const (
+	logFormatFlagName   = "log-format"
+	metricsPortFlagName = "metrics-port"
+	portFlagName        = "port"
+)
+
+func init() {
+	flags := startCmd.Flags()
+
+	// allows environment variables to use _ instead of -
+	flags.Int32P(portFlagName, "p", 8013, "Port to listen on")
```

Comment:
Will the proxy be able to do flag evaluation?

Reply:
At least in this version no, it should only act as a simple proxy for reading configuration changes from FeatureFlagConfigurations without the required permissions

Reply:
Ah, I see. This is the port used for proxying the requests. Got it, thanks!